.bingo/golangci-lint.mod

```diff
@@ -1,7 +1,7 @@
 module _ // Auto generated by https://github.com/bwplotka/bingo. DO NOT EDIT
 
-go 1.22.1
+go 1.23.0
```

Comment:
@tmshort this one will make fail in the check but is OK because we have the version already.